### DISCUSSION TITLE
How do we add image tests?

### DISCUSSION THREAD

**Discussion by rcomer** [SciTools/iris#4802] [Q&A] [answered]
During the course of #4792, I realised I have no idea how to add an image test to Iris.  _Changing_ the reference image [is well documented](https://scitools-iris.readthedocs.io/en/latest/developers_guide/contributing_graphics_tests.html#reviewing-failing-tests) and `idiff.py` sorts out the hash for the image name and json file entry.  If adding a new reference image, do you just do this manually or is there a tool?  Is there some documentation I've missed?

I thought about opening this as a documentation issue but

* we don't add new image tests very often
* we may be changing image testing approach soon

so documenting it right now may be a waste of time.

**Comment by rcomer** [comment]
I see @wjbenfold already has this covered in #4759 👀

**Reply by wjbenfold** [reply]
I've also struggled to work this out previously - it'd be good to know if you think the docs explanation in #4759 make sense :)

**Comment by rcomer** [comment, marked as answer]
Now documented at https://scitools-iris.readthedocs.io/en/latest/developers_guide/contributing_graphics_tests.html#adding-a-new-image-test.
